src/sentry/apidocs/examples/sentry_app_examples.py

```diff
@@ -8,13 +8,15 @@ class SentryAppExamples:
             value={
                 "allowedOrigins": [],
                 "author": "ACME Corp",
-                "avatars": {
-                    "avatarType": "avatar",
-                    "avatarUuid": "6c25b771-a576-4c18-a1c3-ab059c1d42ba",
-                    "avatarUrl": "https://example.com/avatar.png",
-                    "color": False,
-                    "photoType": "icon",
-                },
+                "avatars": [
+                    {
+                        "avatarType": "avatar",
+                        "avatarUuid": "6c25b771-a576-4c18-a1c3-ab059c1d42ba",
+                        "avatarUrl": "https://example.com/avatar.png",
+                        "color": False,
+                        "photoType": "icon",
+                    }
+                ],
```

Comment:
Fix to the avatar serialization while we're here since we actually return a list of avatars (one for logo, another for icon)